src/fetchers/MainPage/MainPage.tsx

```diff
@@ -26,13 +26,20 @@ const recentSearches = ["crypto", "soccer", "soccer"];
 const MainPage: React.FC = () => {
   const [filterType, setFilterType] = useState(FILTER_OPTIONS.TOP);
 
+
   const [filterTabletOn, setFilterTabletOn] = useState(true);
   const DesktopSize = useWindowDimensions();
   
   //fetch data from server
   // const [dataCards, setDataCards] = useState([])
   // const [dataGraphs, setDataGraphs] = useState([]);
 
+  const closeSidebar = () =>{
+    if(filterTabletOn){
```

Comment:
generally the word mobile is better than tablet. I recommend to change it across the project.
change filterTabletOn => filterMobileOpen. its more clear